rosidl_generator_c/msg/DynamicArrayPrimitivesNested.msg

```diff
@@ -0,0 +1 @@
+DynamicArrayPrimitives[] msgs
```

Comment:
"We" still do: 07b54e77544d95b4c3dcb1c9ff76a75e4111ba87

Otherwise e.g. simply calling `cat <filename>` results in not that nice output.